docs/reference/cluster.asciidoc

```diff
@@ -6,23 +6,30 @@
 ["float",id="cluster-nodes"]
 == Node specification
 
-Most cluster level APIs allow to specify which nodes to execute on (for
-example, getting the node stats for a node). Nodes can be identified in
-the APIs either using their internal node id, the node name, address,
-custom attributes, or just the `_local` node receiving the request. For
-example, here are some sample executions of nodes info:
+Most cluster level APIs allow to specify nodes (for example, getting
+the node stats for a certain node). Nodes can be identified in
+the APIs either using their internal node id, node name, address,
+built-in/custom attributes, or `_local`, `_master`, `_all` for the relevant nodes.
+Here are examples of using nodes info API for each case:
 
 [source,js]
 --------------------------------------------------
-# Local
+# Local node
 GET /_nodes/_local
-# Address
+# Master node
+GET /_nodes/_master
+# All nodes
+GET /_nodes/_all
+# Using address
 GET /_nodes/10.0.0.3,10.0.0.4
 GET /_nodes/10.0.0.*
-# Names
+# Using node name
 GET /_nodes/node_name_goes_here
 GET /_nodes/node_name_goes_*
-# Attributes (set something like node.attr.rack: 2 in the config)
+# Using built-in attributes (master, data, ingest or coordinating_only)
+GET /_nodes/master:false,data:false,ingest:true
```

Comment:
This is equivalent to the simpler `GET /_nodes/ingest:true` so I don't think we should have this.